### PR TITLE
Pranita_Assignment_17_Done_45min

### PR DESCRIPTION
**Pull request by PranitaPJun20**
image
counting

### CODE REVIEW

File: src/pranitaPJun20/IndexOfChar.java [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+ */
+package pranitaPJun20;
+
+public class IndexOfChar {
+	
+	void displayFirstLastIndexCharString(String input) {
+		for(int index=0;index<input.length();index++) {
+			char ch = input.charAt(index);			
+			if(index==input.indexOf(ch)) {
+				if (input.indexOf(ch)!=input.lastIndexOf(ch))
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/IndexOfChar.java:22]: Perfect logic.

File: src/pranitaPJun20/StringCount.java [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+		
+		for(int index=0; index<input.length(); index++) {
+			char ch = input.charAt(index);
+			if(Character.isDigit(ch)) 
+				countDigit++;
+			
+			else if(Character.isLetter(ch)) {
+				countLetter++;
+				if(Character.isUpperCase(ch))
+					countUppercase++;
```
Comment (KrishnaKTechnocredits) [inline src/pranitaPJun20/StringCount.java]: else if under> if and else block not required to implement letters logic, Think here. Also countLetter without this variable you can achieve output.

File: src/pranitaPJun20/IndexOfChar.java (hunk 1 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/IndexOfChar.java:22]: Thank you :)

File: src/pranitaPJun20/StringCount.java (hunk 2 above)
Comment (PranitaPJun20) [reply src/pranitaPJun20/StringCount.java]: Yes krishna.Got it.Will update and commit

Comment (PranitaPJun20) [reply src/pranitaPJun20/StringCount.java]: Changes done.Please review

**Pull request closed without merging**